src/main/resources/db/mysql/petclinic_db_setup_mysql.txt

```diff
@@ -23,7 +23,15 @@
 
 3) Run the app with `spring.profiles.active=mysql` (e.g. as a System property via the command
    line, but any way that sets that property in a Spring Boot app should work).
+   
+4) For newer versions of Spring Boot, those will fork the JVM that executes the webserver,
```

Comment:
I don't think a new item should be added, considering that setting the version always worked. It just happend to work with only `-D` if you don't fork the process.

Reply:
I reworded this into step 3.